lib/arclight/custom_document.rb

```diff
@@ -31,5 +34,34 @@ def to_solr(solr_doc = {})
     def names
       [corpname, famname, name, persname].flatten.compact.uniq - repository
     end
+
+    # Combine subjets into one group from:
+    #  <controlaccess/><subject></subject>
+    #  <controlaccess/><function></function>
+    #  <controlaccess/><genreform></genreform>
+    #  <controlaccess/><occupation></occupation>
+    def access_subjects(subjects = [])
```

Comment:
Something that might be a bit more idiomatic would be to not have the method take any arguments and assign a local `subjects` variable in the method to be the results of the `search` function coerced into an array (using `to_a`).

That would make the like 44 look something like
```ruby
subjects = search("//*[local-name()...").to_a
```